sdk/core/core-tracing/test/traceParentHeader.spec.ts

```diff
@@ -1,125 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import { assert } from "chai";
-import { extractSpanContextFromTraceParentHeader, getTraceParentHeader } from "../src";
-import { TraceFlags, SpanContext } from "@opentelemetry/api";
-
-describe("traceParentHeader", () => {
-  describe("#extractSpanContextFromTraceParentHeader", () => {
-    it("should extract a SpanContext from a properly formatted traceparent", () => {
```

Comment:
should some of these tests get ported over to `instrumenter.spec.ts`?

Reply:
We definitely will need to port them over, but there's no implementation logic in core-tracing anymore. So the idea here is to port them over when we add the core-tracing-opentelemetry package as part of #18651 (I added a comment to make sure we remember to port these over)